spec/money/arithmetic_spec.rb

```diff
@@ -482,7 +482,7 @@
       end
     end
 
-    context "with infinite_precision", :infinite_precision do
+    context "with infinite_precision", :default_infinite_precision do
```

Comment:
This one (and two above) should be `:default_infinite_precision_true` instead, right? ^